matlab_simulation/code v1.0/ME597-6-FastSLAM/newCode/ekf_FS1.m

```diff
@@ -0,0 +1,34 @@
+function [mup,Spp,Hmu]=ekf_FS1(Xp,y,mu,S,Qi,check)
```

Comment:
Again, I'm confused why fast slam stuff was updated in the pull request for issue #3 Gyro covariance estimation. The rest of the files in this pull request are all related to SLAM so I'm going to assume they are here for a reason and just carry on. 

Anyway, each function should begin with a block of comments to explain the function and inputs/outputs of the functions. Please see the MatLab style guide section on function comments:
[https://sites.google.com/site/matlabstyleguidelines/layout-documentation/comments#TOC-Function-header-comments-should-support-the-use-of-help-and-lookfor](https://sites.google.com/site/matlabstyleguidelines/layout-documentation/comments#TOC-Function-header-comments-should-support-the-use-of-help-and-lookfor)